Pod/Classes/Aerodramus.m

```diff
@@ -179,7 +175,6 @@ - (NSDictionary *)mapDict:(NSArray *)array map:(id (^)(id object))block {
 - (void)performRequest:(NSURLRequest *)request completionHandler:(void (^)(NSData *data, NSURLResponse *response, NSError *error))completionHandler;
 {
     NSURLSessionConfiguration *configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
-    configuration.requestCachePolicy = NSURLRequestReloadIgnoringCacheData;
```

Comment:
Oops, looks like this got removed, my fault. Will re-add in another PR. /cc @sarahscott